server/mrt_converter/mrt_converter.go

```diff
@@ -112,3 +118,56 @@ func parseUpdate(collector string, h *mrt.MRTHeader, buf []byte) (*update, error
 		Attributes: translateAttrs(bgpUpdate.PathAttributes),
 	}, nil
 }
+
+// convert translates the bzip'ed MRT raw bytes into a BigQuery compatible
+// format and write to GCS.
+func convert(collector string, src []byte, dst io.Writer) error {
+	br := bzip2Reader(bytes.NewReader(src))
+	gr := gzip.NewWriter(dst)
```

Comment:
this might make more sense as 'gw', but 'gr' is probably also ok:)
style/etc call for you to decide on.

Reply:
Yes - "gw" is a better name. 

btw, I got some pings for internal changelists, and someone told me not to use global var any more, so I use dependency injection here instead.